src/Model.php

```diff
@@ -1173,20 +1173,20 @@ public function reload()
      * record in the database.
      *
      * @param mixed $newId
-     *
-     * @return $this
      */
-    public function duplicate($newId = null)
+    public function duplicate($newId = null): self
```

Comment:
annotate with `@return static` and remove `: self`, otherwise `static` return type is not recognized by most IDEs